salt/modules/zfs.py

```diff
@@ -908,67 +929,57 @@ def hold(tag, *snapshot, **kwargs):
         specifies that a hold with the given tag is applied recursively to
         the snapshots of all descendent file systems.
 
-    .. note::
+    .. versionadded:: 2016.3.0
+    .. versionchanged:: 2018.3.1
 
-        A comma-separated list can be provided for the tag parameter to hold multiple tags.
+    .. warning::
+
+        As of Fluorine the tag parameter no longer accepts a comma-separated value.
```

Comment:
We should probably update the Fluorine references, there are a few around I think.

Reply:
Already went through and changed the Fluorine ref's that applied to versionchanged, but left the ones about things deprecating 2 releases later etc.  Also corrected spelling of Flourine to Fluorine.

Are there a few places other than those were is would be appropriate to change Fluorine to 2018.3.1 that I missed ?

Reply:
This warning will already apply to 2018.3.1 so we should probably update at least this reference. I think I saw a few more. ~~Let me look for them again.~~ looks like it was just this one.